addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -819,6 +819,19 @@ body.editor_enable.editor_has_snippets {
                 }
             }
         }
+        
+        // Background position overlay
+        &.o_background_position_overlay {
+            background-color: rgba(0,0,0,.7);
+
+            .o_overlay_content {
+                cursor: grab;
+
+                .grabbing {
```

Comment:
Even this one should start with `o_` and in fact, all should start with `o_we_` since it is in web_editor. I'll change that.